src/renderers/shared/fiber/ReactFiberDevToolsHook.js

```diff
@@ -35,7 +35,22 @@ if (
 
   injectInternals = function(internals: Object) {
     warning(rendererID == null, 'Cannot inject into DevTools twice.');
-    rendererID = inject(internals);
+    // A dummy function to check for minification during runtime
+    // Refer https://github.com/facebook/react-devtools/issues/694#issuecomment-300535376
+    var testMinification = function testMinification() {
+      if (__DEV__) {
+        return 42;
+      }
+    };
+    warning(
```

Comment:
Move `testMinification` out of `injectInternals`? Do we need it if there DevTools are not installed?